MobileCoin.podspec

```diff
@@ -58,7 +58,7 @@ Pod::Spec.new do |s|
       "Sources/Network/*.{h,m,swift}",
```

Comment:
update line:6 `s.version = "1.2.4"` -> `s.version = "2.0.0"`

and update the changelog.md

Reply:
done

Reply:
(done in PR188)